src/structs/jpeg_header.rs

```diff
@@ -848,6 +881,11 @@ impl JPegHeader {
             }
         }
 
+        // calculated preshifted value for each code since the length is encoded in the bottom 4 bits
```

Comment:
Can it be a part of init_fast_lookups?

Reply:
you're right... but even better this code can just be deleted